tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1862,6 +1862,56 @@ error: Cannot instantiate type \\"BucketProps\\" because it is a struct and not
 
  
  
+Tests 1 failed (1)
+Test Files 1 failed (1)
+Duration <DURATION>"
+`;
+
+exports[`super_call.w 1`] = `
+"error: Cannot call super method because we're not in an instance method (\\"super\\" is unknown in this context)
+   --> ../../../examples/tests/invalid/super_call.w:32:7
+   |
+32 | super.do();
+   |       ^^ Cannot call super method because we're not in an instance method (\\"super\\" is unknown in this context)
```

Comment:
Is it possible to simplify this error message to something like:
```suggestion
   |       ^^ Cannot call super method inside of a static method
```

Reply:
There's also the case of calling a `super` method from the global (top level) context. Anyway I'll shorten it :+1: